src/containers/App.jsx

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import "./index.css";
 import { Switch, Route, BrowserRouter as Router } from "react-router-dom";
 import { Home } from "../components/home/Home";
-import { About } from "../components/about/About";
+import { About } from "../components/About";
```

Comment:
Why didn't you use the "about" folder only and instead of that you created a new file in "components" folder

Reply:
i didn't create a new one (: it was there at the init. of files which i believed shouldn't be there. for no duplication i improvised on the file names and used it both. if we are going to use the inner one i will delete the outer one.